test/graphql/mutations/associate_systems_test.rb

```diff
@@ -71,4 +71,35 @@ class AssociateSystemsMutationTest < ActiveSupport::TestCase
 
     assert_equal(profiles(:one).hosts.pluck(:id), [NEW_ID])
   end
+
+  test 'removes systems from a profile' do
+    query = <<-GRAPHQL
```

Comment:
New test! This associateSystems mutation actually changes slightly from the way it used to work. The array of systems passed in now will become the systems associated with the profile, no more, no less, regardless of if the profile already has systems associated. Those systems not specified in the mutation will be removed from the profile.